sips/sip-submission.md

```diff
@@ -1,61 +1,279 @@
-* `Not Accepted` - The SIP committee feels the SIP is not needed for the Scala language and turns down the proposal, with an explanation of why.
+The SIP process involves the following parties:
 
-## What happens for a Scala major release? ##
+1. The SIP Authors
+2. The Process lead
+3. The SIP Committee
 
-Before a Scala release, the committee will make a final decision for each SIP whether it should be accepted, rejected, or delayed for the next release. Accepted SIPs will be rolled into an upcoming Scala release and placed in the accepted folder.  Rejected SIPs will be left in the SIP repository under the "rejected sips" section.  Delayed SIPs will remain pending.
+### The SIP Authors
 
+Authors are responsible for building consensus within the community and
+documenting dissenting opinions before the SIP is officially discussed by the
+SIP committee. Their goal is to convince the committee that their proposal is
+useful and addresses pertinent problems in the language as well as interactions
+with already existing features. Authors can change over the lifecycle of the
+SIP.
 
-## Who is on the SIP committee ##
+### The Process lead
 
-Right Now:
+The Process lead is the responsible of the smooth running of SIPs and SLIPs. He
+or she appoints the committee members, calls the meetings monthly, assigns new
+proposals to the members, and ensures that all of them are discussed within a
+short period of time.
 
-* Martin Odersky
-* Josh Suereth
-* Adriaan Moors
-* Seth Tisue
-* Dick Wall
-* Heather Miller
+### The SIP Committee
+
+The SIP Committee is an experienced group of people with knowledge of the
+compiler internals, responsible for the strategic direction of Scala. Members
+are tasked with (a) communicating with the community, (b) weighing in pros and
+cons of every proposal, and (c) accepting, postponing or rejecting the proposal.
+
+Committee members should be either individuals responsible for a specific part
+of the Scala codebase, committers or contributors of the Scala compiler.
+Exceptionally, members may also be important representatives of the community
+with a high technical knowledge to understand the implications of every proposal
+and participate into the discussions. The members are elected by the Process
+Lead based on their expertise and implication in the community.
+
+The current committee members are:
+
+- Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+- Adriaan Moors ([@adriaanm](https://github.com/adriaanm)), Lightbend
+- Erik Osheim ([@non](https://github.com/non)), independent
+- Heather Miller ([@heathermiller](https://github.com/heathermiller)), Scala Center
+- Seth Tisue ([@SethTisue](https://github.com/SethTisue)), Lightbend
+- Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), EPFL
+- Eugene Burmako ([@xeno-by](https://github.com/xeno-by)), EPFL
+- Andrew Marki ([@som-snytt](https://github.com/som-snytt)), independent
+- Josh Suereth ([@jsuereth](https://github.com/jsuereth)), Google
```

Comment:
@jsuereth I wonder if you would prefer “independent“ here, since you aren't really “representing” Google?